core/arch/arm/include/sm/optee_smc.h

```diff
@@ -559,6 +559,37 @@
 /* See OPTEE_SMC_CALL_WITH_REGD_ARG above */
 #define OPTEE_SMC_FUNCID_CALL_WITH_REGD_ARG	U(19)
 
+/*
+ * Access to the secure watchdog.
+ *
+ * Call requests usage:
+ * a0	SMC Function ID, OPTEE_SMC_FUNCID_WATCHDOG
+ * a1	Watchdog command
+ *	0 : Init watchdog
+ *	1 : Set timeout
+ *	2 : Enable watchdog
+ *	3 : Ping the watchdog for refresh
+ *	4 : Get time left
+ * a2-6	Not use
```

Comment:
Wrong. a2 is used to set the timeout when a0 == 1 and to enable/disable the watchdog when a0 == 2.
s/use/used/

Reply:
Agree, I'll add it.